bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/auxfile/AuxFileEditor.scala

```diff
@@ -27,7 +27,7 @@ object AuxFileEditor {
       lab.text = size.toString
   })
 
-  private val Format = new SimpleDateFormat("MM/dd/yy HH:mm:ss") {
+  private val Format = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss") {
```

Comment:
It was unexpectedly easy 🍰 

<img width="439" alt="sync_history" src="https://user-images.githubusercontent.com/215438/26840556-7bb8799c-4ab4-11e7-87e9-98b0c3919ca6.png">